pkg/threescale/threescale.go

```diff
@@ -307,8 +311,12 @@ func parseURL(url *url.URL) (string, string, int) {
 // rpcStatusErrorHandler provides a uniform way to log and format error messages and status which should be
 // returned to the user in cases where the authorization request is rejected.
 func rpcStatusErrorHandler(userFacingErrMsg string, fn func(string) rpc.Status, err error) (rpc.Status, error) {
+	var errMsg string
+	if err != nil {
+		errMsg = err.Error()
+	}
 	if userFacingErrMsg != "" {
-		err = fmt.Errorf("%s - %s", userFacingErrMsg, err.Error())
+		err = fmt.Errorf("%s - %s", userFacingErrMsg, errMsg)
```

Comment:
why this change?

Reply:
if the provided err is nil it would cause a panic here

Reply:
that's right, but `errMsg` is not used anywhere else - so it either is empty or it has something useful, but still being printed here - so why not move the above check on `err` not being nil inside this block where some error will be displayed?